ConsumerApi.Tests.Integration/Features/Messages/POST.feature

```diff
@@ -0,0 +1,18 @@
+﻿@Integration
+Feature: POST Message
+
+User sends a Message
+
+Scenario: Sending a Message
+	Given Identities i1 and i2 with an established Relationship
+	When a POST request is sent to the /Messages endpoint
+	Then the response status code is 201 (Created)
+	And the response contains a CreateMessageResponse
+
+Scenario: Sending a Message to Identity to be deleted
+	Given Identities i1 and i2 with an established Relationship
+	And Identity i2 is to be deleted
```

Comment:
I think I would become slightly more technical here in order to be more clear.

Also, you don't need to repeat "Identity" here, as `i2` is properly defined in the first step.

```suggestion
	And i2 is in status "ToBeDeleted"
```